tests/Bridge/Symfony/PropertyInfo/Metadata/Property/PropertyInfoPropertyNameCollectionFactoryTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Tests\Bridge\Symfony\PropertyInfo\Metadata\Property;
+
+use ApiPlatform\Core\Bridge\Symfony\PropertyInfo\Metadata\Property\PropertyInfoPropertyNameCollectionFactory;
+use ApiPlatform\Core\Tests\Fixtures\DummyObjectWithoutPublicProperties;
+use ApiPlatform\Core\Tests\Fixtures\DummyObjectWithPublicProperties;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\PropertyInfo\PropertyInfoExtractor;
+
+/**
+ * @author Oskar Stark <oskarstark@googlemail.com>
+ */
+class PropertyInfoPropertyNameCollectionFactoryTest extends TestCase
+{
+    public function testCreateMethodReturnsEmptyPropertyNameCollection()
+    {
+        $factory = new PropertyInfoPropertyNameCollectionFactory(new PropertyInfoExtractor());
```

Comment:
Apologies for the rather big diff posted below - the tests are currently failing because we need to inject at least one list extractor:

```suggestion
        $factory = new PropertyInfoPropertyNameCollectionFactory(new PropertyInfoExtractor([
            new ReflectionExtractor(),
        ]));
```